cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,71 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() =>{
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
   });
-});
+
+  it('Assert the pagination and rows count selection', () => {
+    cy.get('.pagination-bottom').should('exist');
+    cy.get('[aria-label="rows per page"]').should('exist');
+    cy.get('[aria-label="rows per page"]').select('5 rows');
+  });
+
+  it('Should add a new worker and delete him', () => {
+    cy.get('#addNewRecordButton.btn.btn-primary').click();
+    cy.findByPlaceholder('First Name').type(user.firstName);
+    cy.findByPlaceholder('Last Name').type(user.lastName);
+    cy.findByPlaceholder('name@example.com').type(user.email);
+    cy.findByPlaceholder('Age').type(user.age);
+    cy.findByPlaceholder('Salary').type(user.salary);
+    cy.findByPlaceholder('Department').type(user.department);
+    cy.get('#submit').click();
+    cy.get('#delete-record-4').click();
+  });
+
+  it('Should delete all workers', () => {
+    cy.get('#delete-record-1').click();
+    cy.get('#delete-record-2').click();
+    cy.get('#delete-record-3').click();
+    cy.get('.rt-noData').should('exist');
+  });
+
+  it('Should find a worker in the search field and edit it', () => {
+    cy.findByPlaceholder('Type to search').type('Cierra');
+    cy.get('#edit-record-1').click();
+    cy.findByPlaceholder('Last Name').type(user.lastName);
+    cy.get('#submit').click();
+    cy.get('.rt-td').should('contain', user.lastName);
+  });
+
+  it('Assert the search by all column values', () => {
+    cy.findByPlaceholder('Type to search').type('Cierra');
```

Comment:
you use string 'Cierra' at least 5 times. Make a constant for it and for all values that you use 2 times and more